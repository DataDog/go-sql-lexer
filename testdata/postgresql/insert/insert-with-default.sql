INSERT INTO products (name, price, description) VALUES ('New Product', 123, DEFAULT);
