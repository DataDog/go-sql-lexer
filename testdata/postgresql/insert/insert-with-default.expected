[
    {
        "expected": "INSERT INTO products ( name, price, description ) VALUES ( ?, DEFAULT )",
        "statement_metadata": {
            "size": 14,
            "tables": [
                "products"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]